source/agora/consensus/data/Transaction.d

```diff
@@ -60,6 +60,17 @@ public struct Transaction
     /// The data to store
     public DataPayload payload;
 
+    /// The size of the Transaction object
+    public ulong sizeInBytes() const nothrow pure @safe @nogc
+    {
+        ulong size = type.sizeof + payload.sizeInBytes();
+        foreach (const ref input; inputs)
```

Comment:
Suggested change.
`this.inputs`

Reply:
kind of contraversial :) If we modify the the style guide that we have to prefix every class variable with this then I am happy to comply. VSCode uses different colors for class members vs local variables though.

Reply:
It's not in the style guide ? Because we've always done it. Indeed should add it.

Reply:
> It's not in the style guide ? Because we've always done it. Indeed should add it.

Yeah, I just checked it is not there. I will go through this PR and add 'this' and also change the style guide